resolwe_bio/processes/reads_processing/cutadapt.yml

```diff
@@ -115,6 +121,13 @@
           default: false
           description: |
             Interpret IUPAC wildcards in reads.
+        - name: no_indels
+          label: No indels
+          type: basic:boolean
+          required: false
```

Comment:
If the default is set, you should omit the `required: false`, as it is redundant.